### PR TITLE
Non-linaer regression

### PR DESCRIPTION
**Pull request by jan-www**

### CODE REVIEW

File: src/model/non-linear-model/NonLinearRegression.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import { Rank, Tensor, scalar, variable, Tensor1D, RecursiveArray, Variable, Scalar, sub, add, gather, stack, tidy, dispose } from '@tensorflow/tfjs-core';
```
Comment (yorkie) [inline src/model/non-linear-model/NonLinearRegression.ts:1]: I think we need to unify our naming convention, see https://github.com/imgcook/datacook/blob/ce426da8c927d71fb5fb5425653603447c8a55be/src/model/linear-model/linear-regression.ts.

**Review by yorkie — CHANGES_REQUESTED** [review]

File: src/model/non-linear-model/NonLinearRegression.ts (hunk 1 above)
Comment (jan-www) [reply src/model/non-linear-model/NonLinearRegression.ts:1]: The inconsistency is due to the method used to optimize non linear model is not stochastic gradient descent

File: src/linalg/jacobian.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { Scalar, Tensor, Variable, variableGrads, gather, stack, tidy } from "@tensorflow/tfjs-core";
```
Comment (yorkie) [inline src/linalg/jacobian.ts:1]:
```suggestion
import { Scalar, Tensor, Variable, variableGrads, gather, stack, tidy } from '@tensorflow/tfjs-core';
```

File: src/linalg/jacobian.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import * as tf from '@tensorflow/tfjs-core';
```
Comment (yorkie) [inline src/linalg/jacobian.ts:2]: Duplicated import for tfjs-core.

**Review by yorkie — APPROVED** [review]

**Pull request merged**